docs/makepages.py

```diff
@@ -100,7 +100,14 @@
       <Route path="arduinoservo" component={Arduino101ServoPage} />
       <Route path="arduinosoundrecorder" component={Arduino101SoundRecorderPage} />
     </Route>
-    <Route path="faq">
+    <Route path="linkit">
+	  <Route path="linkitintro" component={linkitintroPage} />
```

Comment:
After compiling, the page doesn't load here due to capitalization mismatch between the `...Page` components and the file names in the pages directory.